packages/webpack.config.js

```diff
@@ -33,7 +33,7 @@ function getConfig(pkg) {
     module: {
       rules: [
         {
-          test: /\.tsx?$/,
+          test: /\.tsx?$/u,
```

Comment:
`u` is for unicode. Using it on every regex is recommended because it enforces a stricter regex syntax.